apps/src/templates/AssignButton.jsx

```diff
@@ -77,9 +81,15 @@ class AssignButton extends React.Component {
     const {confirmationDialogOpen} = this.state;
     const {assignmentName, sectionName} = this.props;
 
+    // Adjust styles if locale is RTL
+    const isRtl = getStore().getState().isRtl;
```

Comment:
Instead of calling `getStore` here, add `isRtl: state.isRtl` on line 119 to get the value from the store. `isRtl` will then be accessible as a prop in the component (this is generally the pattern we follow). See `hiddenStageState` for an example.